imap_processing/lo/l1b/lo_l1b.py

```diff
@@ -11,19 +11,19 @@
 
 def lo_l1b(dependencies: dict, data_version: str):
     """
-    Process IMAP-Lo L1A data into L1B CDF data products.
+    Will process IMAP-Lo L1A data into L1B CDF data products.
 
     Parameters
     ----------
     dependencies : dict
-        dictionary of datasets needed for L1B data product creation in xarray Datasets.
+        Dictionary of datasets needed for L1B data product creation in xarray Datasets.
     data_version : str
-        Version of the data product being created
+        Version of the data product being created.
 
     Returns
     -------
-    created_file_paths: list[Path]
-        location of created CDF files
+    created_file_paths : list[Path]
```

Comment:
Similarly, this might need to be pathlib.Path
```suggestion
    created_file_paths : list[pathlib.Path]
```